### PR TITLE
feature: Opcode Logical And of byte with Register A into Register A

### PR DESCRIPTION
**Pull request by Gvillat**

### CODE REVIEW

File: crates/cpu/src/opcodes/data/logical.rs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+use crate::Flags;
+
+// const MAX_BIT3: usize = (1 << 4) - 1;
```
Comment (cempassi) [inline crates/cpu/src/opcodes/data/logical.rs]: Pas de commentaire sans bonne raison

File: crates/cpu/src/opcodes/data/logical.rs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+pub fn and(first: u8, second: u8) -> u16 {
+    let data = first & second;
```
Comment (cempassi) [inline crates/cpu/src/opcodes/data/logical.rs]: Pareil que le OR, dst et src, et le sens de l'operation

**Review by cempassi — CHANGES_REQUESTED** [review]

File: crates/cpu/src/opcodes/data/logical.rs (hunk 1 above)
Comment (Gvillat) [reply crates/cpu/src/opcodes/data/logical.rs]: mistake were mades

File: crates/cpu/src/opcodes/data/logical.rs (hunk 2 above)
Comment (Gvillat) [reply crates/cpu/src/opcodes/data/logical.rs]: ok

**Pull request closed without merging**